src/models/api.js

```diff
@@ -24,10 +24,10 @@ export async function getCaptures() {
     const res = await axios.get(url);
     const data = await res.data;
     log.warn('url:', url, 'took:', Date.now() - begin);
-    return data.captures;
+    return data.captures || [];
   } catch (err) {
-    log.error(err.message);
-    throw err;
+    log.error('Error in getCaptures:', err.message);
+    return [];
```

Comment:
@yunchipang why you disable the throwing here?

Reply:
because if it throws an error, `getCaptures()` will not return anything, then `captures` in static props gets undefined, and the entire top page just shows 404.

for now, to inform the error i'm just logging it. would you like this to be handle in some other way? my purpose is to have some props display on the page even when some other props were not populated properly.

Reply:
I'm not sure, let's merge your code first, and we can continue, we can back to this if there are some problem. 

BTW, I also don't like the try-catch way to handle error, I prefer to use Promise function to do it:

promise
.then(goodHandler)
.catch(badHandler)